zebra-consensus/src/transaction/tests.rs

```diff
@@ -641,7 +641,7 @@ fn v4_transaction_with_conflicting_sprout_nullifier_across_joinsplits_is_rejecte
         };
 
         // Sign the transaction
-        let sighash = transaction.sighash(network_upgrade, HashType::ALL, None);
+        let sighash = transaction.sighash(network_upgrade, HashType::ALL, &Vec::new(), None);
```

Comment:
Suggestion: same as above:

```suggestion
        let sighash = transaction.sighash(network_upgrade, HashType::ALL, &[], None);
```

Reply:
Done in b36577315 / #3446